src/rules/jsdocFormatRule.ts

```diff
@@ -85,6 +97,22 @@ function walk(ctx: Lint.WalkContext<Options>) {
             return;
         }
 
+        // This is a single-line comment using the multiline format
+        if (ctx.options.forceSingleline && lines.length === 3) {
+            // Add 3 to account for the leading /**
+            const endOfFirstLine = pos + firstLine.length + 3;
+            const endOfSecondLine = endOfFirstLine + lines[1].length + 1;
+            const endOfThirdLine = endOfSecondLine + lines[2].length + 1;
```

Comment:
this logic feels brittle, but i don't have alternatives to offer. would it handle trailing whitespace?

Reply:
Yea, it's quite brittle. I misunderstood exactly what `pos` and `fullText` referred to when I wrote this. (I'd been under the impression that `fullText` was the lines containing the comment block and `pos` was, effectively, the indent.) 

I actually ended up rewriting this addition in isolation from the rest of `jsdoc-format` so we could use it here until this PR is ready to go. In so doing, I gave up on a per-line marker and just mark the whole comment block which seems a lot more stable.

Reply:
@ianwremmel would you kindly update this PR with your new implementation? be the change you want to see in the repo!

Reply:
haha, yea, hopefully today or tomorrow; work priorities shifted a bit and it's been hard to find the time.